docker-compose.yml

```
@@ -24,9 +21,6 @@ services:
             - type: bind
               source: ./database
               target: /var/lib/mysql
-            - type: bind
-              source: ./certs
-              target: /etc/certs/
```

```suggestion
            - type: bind
              source: ./certificates
              target: /etc/certs/
```
Because i think, it 's simpler to do that like this !